src/Hl7.Fhir.Base/Specification/Terminology/CodeSystemTerminologyService.cs

```diff
@@ -0,0 +1,175 @@
+#nullable enable
+
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Rest;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using T = System.Threading.Tasks;
+
+namespace Hl7.Fhir.Specification.Terminology
+{
+    /// <summary>
+    /// Base class for checking Code terminology
+    /// </summary>
+    abstract public class CodeSystemTerminologyService : ITerminologyService
```

Comment:
This is really minor, but this is usually `public abstract` instead of the other way around.

Reply:
And does this class have to be public? Isn't internal good enough? Or do we want our users also to use this?

Reply:
It has to be public or the code will not compile, derived classes cannot be more accessible than their base class